### PR TITLE
#1239 create save file modal window

### PR DESCRIPTION
**Pull request by ViktoriiaFedotkina**

### CODE REVIEW

**Review by ElenaOdnoshivkina — APPROVED** [review]

File: packages/ketcher-polymer-editor-react/src/components/shared/inputField/InputField.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  '&:active, &:focus': {
+    color: theme.ketcher.color.input.text.active
+  },
+
+  '&:hover': {
+    backgroundColor: theme.ketcher.color.input.background.hover
+  }
+}))
+
+export const InputField = ({ value, id, onChange, label, ...rest }: Props) => {
```
Comment (karen-sarkisyan) [inline packages/ketcher-polymer-editor-react/src/components/shared/inputField/InputField.tsx:51]: `InputField` name seems a little too generic for this. Maybe let's call it `TextInputField`?

**Review by karen-sarkisyan — CHANGES_REQUESTED** [review]

File: packages/ketcher-polymer-editor-react/src/theming/defaultTheme/theme.types.ts [hunk 2]
```diff
@@ -77,10 +78,22 @@ export type EditorTheme = {
     divider: string
     error: string
     input: {
+      text: {
+        rested: string
```
[the remark this answers is not in the document]
Comment (ViktoriiaFedotkina) [reply packages/ketcher-polymer-editor-react/src/theming/defaultTheme/theme.types.ts:82]: well they are now used in figma. Inactive is disabled, rested is not active but is not disabled as well

File: packages/ketcher-polymer-editor-react/src/components/shared/actionButton/ActionButton.tsx [hunk 3]
```diff
@@ -55,7 +56,7 @@ export const ActionButton = ({
       role="button"
       {...rest}
     >
-      {label}
+      {children || label}
```
[the remark this answers is not in the document]
Comment (ViktoriiaFedotkina) [reply packages/ketcher-polymer-editor-react/src/components/shared/actionButton/ActionButton.tsx:59]: I used this here [packages/ketcher-polymer-editor-react/src/components/modal/save/Save.tsx](https://github.com/epam/ketcher/pull/1246/files#diff-58b0b1e585a5199bccbe910f5482833ffdbd311062a5f73b3d05dda47e1c49fb) (192)

[the remark this answers is not in the document]
Comment (karen-sarkisyan) [reply packages/ketcher-polymer-editor-react/src/components/shared/actionButton/ActionButton.tsx:59]: Oh, right, didn't see it. Fair enough 👍

File: packages/ketcher-polymer-editor-react/src/theming/defaultTheme/theme.types.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (karen-sarkisyan) [reply packages/ketcher-polymer-editor-react/src/theming/defaultTheme/theme.types.ts:82]: You're right. Still, it seems incorrect. I've discussed this with our colleagues, they'll rename input states soon.

**Review by karen-sarkisyan — APPROVED** [review]

**Review by AndreiMazol — APPROVED** [review]

**Pull request merged**